src/Dropdown/Dropdown.tsx

```diff
@@ -72,8 +72,12 @@ export const Dropdown: React.FC<DropdownProps> = ({
   }, [])
 
   const position = useMemo(() => {
-    if (refs.parent.current) {
-      return makeCSS(positioning, { height: refs.parent.current.getBoundingClientRect().height, width: refs.parent.current.getBoundingClientRect().width })
+    if (refs.parent.current && refs.target.current) {
+      return makeCSS(
+        positioning,
+        { height: refs.target.current.getBoundingClientRect().height, width: refs.target.current.getBoundingClientRect().width },
```

Comment:
here there a new parameter that takes height and width from the target component

Reply:
Lets wait until PR with useClientRect() has been merged and then imrpove dropdown with that pattern

Reply:
The dropdown was refactored to use useClientRect(), the pull request is open